scripts/project.js

```diff
@@ -55,3 +55,13 @@ $('.project-preview').click(function(){
     }
   });
 });
+
+
+$('header nav').on('click','li',function(){
+  var $scrollTo = $(this).data('link');
+  $target = $('section[data-content="' + $scrollTo + '"]');
+  console.log($target);
+  $('html, body').animate({
+    scrollTop: $target.offset().top
```

Comment:
I get a console error on this line: Uncaught TypeError: Cannot read property 'top' of undefined

It seems to work, but check that offset() is something you can use there.